AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_editor_utils.py

```diff
@@ -3,281 +3,28 @@
 """
 
-import os
-import sys
-import time
-import azlmbr.atom
-import azlmbr.atomtools as atomtools
-import azlmbr.materialeditor as materialeditor
-import azlmbr.math as math
 import azlmbr.bus as bus
+import azlmbr.materialeditor as materialeditor
 
 
-def is_close(actual, expected, buffer=sys.float_info.min):
-    """
-    :param actual: actual value
-    :param expected: expected value
-    :param buffer: acceptable variation from expected
-    :return: bool
-    """
-    return abs(actual - expected) < buffer
-
-
-def compare_colors(color1, color2, buffer=0.00001):
-    """
-    Compares the red, green and blue properties of a color allowing a slight variance of buffer
-    :param color1: first color to compare
-    :param color2: second color
-    :param buffer: allowed variance in individual color value
-    :return: bool
-    """
-    return (
-            is_close(color1.r, color2.r, buffer)
-            and is_close(color1.g, color2.g, buffer)
-            and is_close(color1.b, color2.b, buffer)
-    )
-
-
-def open_material(file_path):
-    """
-    :return: uuid of material document opened
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "OpenDocument", file_path)
-
-
-def is_document_open(document_id):
-    """
-    :return: bool
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "IsDocumentOpen", document_id)
-
-
-def save_document(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "SaveDocument", document_id)
-
-
-def save_document_as_copy(document_id, target_path):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(
-        bus.Broadcast, "SaveDocumentAsCopy", document_id, target_path
-    )
-
-
-def save_document_as_child(document_id, target_path):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(
-        bus.Broadcast, "SaveDocumentAsChild", document_id, target_path
-    )
-
-
-def save_all():
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "SaveAllDocuments")
-
-
-def close_document(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseDocument", document_id)
-
-
-def close_all_documents():
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseAllDocuments")
-
-
-def close_all_except_selected(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseAllDocumentsExcept", document_id)
-
-
-def get_property(document_id, property_name):
+def get_property(document_id: azlmbr.math.Uuid, property_name: str) -> azlmbr.math.Uuid:
```

Comment:
is this return property value type Any rather than azlmbr.math.Uuid?
does the return need to be checked for None or a specific Invalid value to determine if the call failed.

Reply:
changed to any